cli/app.go

```diff
@@ -2935,6 +2941,16 @@ This won't work unless you have an existing installation of our GitHub app on yo
 	# Restart a module running on your local viam server, by name, without building or reconfiguring.
 	viam module reload --restart-only --id viam:python-example-module
 
+	# Use cloudbuild to build the tar.gz that will be copied to the part for hot reloading.
+	viam module reload --cloud-build
+
+	# Run viam module reload on a mac and use the downloaded viam.json file instead of --part-id
+	viam module reload --cloud-config ~/Downloads/viam-mac-main.json
```

Comment:
the flags below make it seem like you need to pass viam.json as a `--module` arg (unless I'm misreading)

this example is passing the json path as a positional arg